ckanext/datavicmain/actions.py

```diff
@@ -119,4 +123,17 @@ def datavic_user_create(context, data_dict):
         )
 
     log.debug('Created user {name}'.format(name=user.name))
-    return user_dict
\ No newline at end of file
+    return user_dict
+
+def nominate_resource_view(context, data_dict):
+    package_id = _get_or_bust(data_dict, 'package_id')
+    view_id = _get_or_bust(data_dict, 'view_id')
+    pkg_dict = get_action('package_show')(
+        dict(context, return_type='dict'),
```

Comment:
`return_type='dict'`? What is the purpose of this parameter?

Reply:
I don't think you are right. Can you show some proof that `package_show` returns an object if `return_type='dict'` is missing? Maybe link to the code or documentation?

Reply:
You gave me a link to a line, that creates a dictionary. I asked you to give me **some proof that package_show returns an object if return_type='dict' is missing**